### DISCUSSION TITLE
WebSotires Alguém já implementou com React/NextJS/GatsbyJS? Dicas?

### DISCUSSION THREAD

**Discussion by Rychillie** [frontendbr/forum#1972] [Perguntas e Respostas]
Li um artigo sobre o [WebStories do Google](https://stories.google/intl/pt-BR_ALL/) e achei interessante para a criação de conteúdos diferenciados, inclusive ja vi alguns de noticiais e acredito que agrega para o usuário.

A fins de estudo, alguém poderia me recomendar algum conteúdo mais direcionado como artigos?

Estou pedindo ajuda pois nunca trabalhei com Google AMP.

**Comment by felquis** [comment]
Tem duas partes importantes, a parte de projetar o conteúdo https://stories.google/intl/pt-BR_ALL/create/ 

E a parte de integrar com HTML das páginas e neste caso é através do AMP, neste link tem um passo a passo de como usar stories no AMP https://amp.dev/documentation/guides-and-tutorials/start/visual_story/setting_up/?format=stories 

AMP é uma lib, e ela tem algumas convenções e ela prefere que vc use apenas ele idealmente, se for pra começar sugiro começar com HTML normal para ver como que funciona e depois tenta aplica-lo no seu framework favorito.

- https://amp.dev/documentation/guides-and-tutorials/start/create/basic_markup/?format=websites

E relacionado ao nextjs como exemplo:
- https://nextjs.org/docs/api-reference/next/amp
- https://nextjs.org/docs/advanced-features/amp-support/introduction

PS: nunca usei o AMP, mas este é o caminho que eu faria baseado na sua pergunta.
